src/ClientHandle.cpp

```diff
@@ -2568,6 +2586,42 @@ void cClientHandle::SendDisplayObjective(const AString & a_Objective, cScoreboar
 
 
 
+void cClientHandle::SendSetSubTitle(const cCompositeChat & a_SubTitle)
```

Comment:
Not really, because the packet doesn't show the title, it only sets the title and you need to send TitleTimes to actually show the title.